UserInterface/WinForms/LeanWinForm.Extensions.cs

```diff
@@ -23,16 +25,35 @@ namespace QuantConnect.Views.WinForms
     /// Public extensions methods for the forms.
     /// Credit: http://stackoverflow.com/questions/1926264/color-different-parts-of-a-richtextbox-string/1926822#1926822
     /// </summary>
-    public static class RichTextBoxExtensions
+    public static class Extensions
     {
         public static void AppendText(this RichTextBox box, string text, Color color)
         {
-            box.SelectionStart = box.TextLength;
-            box.SelectionLength = 0;
+            box.SafeInvoke(b =>
+            {
+                box.SelectionStart = box.TextLength;
+                b.SelectionLength = 0;
+                box.SelectionColor = color;
+                box.AppendText(text + Environment.NewLine);
+                box.SelectionColor = box.ForeColor;
+                box.ScrollToCaret();
+                return 0;
```

Comment:
If this code needs to be thread safe (in order to prevent color bleed) then we'll need to lock it